test/corectl_integration_test.go

```diff
@@ -171,11 +171,18 @@ func setupTest(t *testing.T, tt test) func(t *testing.T, tt test) {
 		if tt.initTest.teardown == true {
 			t.Log("\u001b[96m *** Teardown *** \u001b[0m")
 
-			args := append(tt.connectString, []string{"app", "rm", "--suppress"}...)
+			args := append(tt.connectString, "app", "ls")
 			cmd := exec.Command(binaryPath, args...)
+			output, err := cmd.CombinedOutput()
+			apps := []map[string]interface{}{}
+			json.Unmarshal(output, &apps)
+			appName := apps[0]["name"].(string)
```

Comment:
I think we need to find a better way to solve this. Right now we delete the first app in the app list regardless the name. Could lead to an unwanted deletion of an app for anyone running the integration tests. Maybe do a diff of the app list between setup and teardown?

Reply:
Been trying some different solutions.

Checking for added apps is not rigorous, since you could have a app with the same name (e.g. from previous tests) which would then cause the tear down no to work and probably tests failing that shouldn't.

The easiest approach is probably to hardcode the name of test apps so that they all use the same name and we just check for that. This could at least be a temporary solution since I removed the functionality to use the properties specified in the config implicitly.
We could at a later stage add support for using config values explicitly for commands like
`app rm` as it seems useful. We just have to figure out how such commands should be written so that the user doesn't remove entities by mistake.

Another approach is to look at the used config and extract the app name from there. It is a workaround but arguably better than hardcoding the app name for tests.

Reply:
I think we should go for the hardcoded test app name for now, unless you have already started implementing reading from config. Going forward we probably will need to add tests for creating e.g. apps with special characters etc and then we can look further into this issue.